### DISCUSSION TITLE
Config / Backup File

### DISCUSSION THREAD

**Discussion by koply** [ppy/osu#20463] [Ideas] [answered]
Re-setting settings and playlists after reinstalling OSU is really a disgrace. Exporting and importing data should definitely be included.

### We need **osu!lazer to osu!lazer** data import.

**Comment by Theighlin** [comment, marked as answer]
You can just save the config file from the previous lazer installation and drop it in the new folder.

**Reply by peppy** [reply]
As above. It's all in a single file you can backup yourself.

**Reply by koply** [reply]
Does it do any autosave?

**Reply by peppy** [reply]
Everything is saved automatically, yes.
